rundown/static/sports.py

```diff
@@ -0,0 +1,43 @@
+"""Module for static sports information used by Rundown."""
```

Comment:
what's the purpose of this being static? We're in the process of adding more sports today (EPL) and will definitely be adding more down the road.

Reply:
I used this static file to allow the user to pass the name of the sport as an alternative to the sport's ID to many of the methods in the `Rundown` class. Just as a convenience for the user so they can avoid remembering sport IDs.

Reply:
Using the sport name is optional - the sport parameter supports both ID and name.

Reply:
Can't we make a request to the sports endpoint and use the `sport_name` to build this list dynamically instead?